runtime/cere/constants/src/lib.rs

```diff
@@ -0,0 +1,78 @@
+// This file is part of Substrate.
```

Comment:
The `runtime/cere/constants/src/lib.rs` file has been removed. You should completely delete it from the PR. All constants are located in `runtime/common/src/constants.rs` file now.